index.test-d.ts

```diff
@@ -0,0 +1,11 @@
+import { expectType } from 'tsd-check';
```

Comment:
Imports, destructuring, and object literals should not have spaces around the identifier. `{foo}`, not `{ foo }`. Just my preference.